vocabularies/UI.xml

```diff
@@ -1401,6 +1401,37 @@ The trend is
         <Annotation Term="Core.IsLanguageDependent" />
       </Term>
 
+      <Term Name="InputMask" Type="UI.InputMaskType" Nullable="false" AppliesTo="Property Parameter">
```

Comment:
"InputMask" threw me off because we already have `Common.Masked` and `Common.MaskedValue` where "masking" means hiding.

Reply:
Good point, suggest calling it `InputPattern` instead.

Reply:
https://www.google.com/search?q=inputmask